lib/src/view/opening_explorer/opening_explorer_screen.dart

```diff
@@ -0,0 +1,845 @@
+                    padding: rowPadding,
+                    child: _WinPercentageChart(
+                      whiteWins: move.white,
+                      draws: move.draws,
+                      blackWins: move.black,
+                    ),
+                  ),
+                ),
+              ],
+            );
+          },
+        ),
+        TableRow(
+          decoration: BoxDecoration(
+            color: moves.length.isEven
+                ? Theme.of(context).colorScheme.surfaceContainerLow
+                : Theme.of(context).colorScheme.surfaceContainerHigh,
+          ),
+          children: [
+            Container(
+              padding: rowPadding,
+              alignment: Alignment.centerLeft,
+              child: const Icon(Icons.functions),
+            ),
+            Container(
+              padding: rowPadding,
+              child: Tooltip(
+                message: '100%',
+                child: Text(formatNum(games)),
+              ),
+            ),
+            Container(
+              padding: rowPadding,
+              child: _WinPercentageChart(
+                whiteWins: whiteWins,
+                draws: draws,
+                blackWins: blackWins,
+              ),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class _GameList extends StatelessWidget {
+  const _GameList({
+    required this.title,
+    required this.games,
+  });
+
+  final String title;
+  final IList<Game> games;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          padding: const EdgeInsets.all(6.0),
+          color: Theme.of(context).colorScheme.primaryContainer,
+          child: Row(
+            children: [
+              Text(title),
+            ],
+          ),
+        ),
+        ...List.generate(games.length, (int index) {
+          return _GameTile(
+            game: games.get(index),
+            color: index.isEven
+                ? Theme.of(context).colorScheme.surfaceContainerLow
+                : Theme.of(context).colorScheme.surfaceContainerHigh,
+          );
+        }),
+      ],
+    );
+  }
+}
+
+class _GameTile extends ConsumerWidget {
+  const _GameTile({
+    required this.game,
+    required this.color,
+  });
+
+  final Game game;
+  final Color color;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    const widthResultBox = 50.0;
+    const paddingResultBox = EdgeInsets.all(5);
+
+    final openingDb = ref.watch(openingExplorerPreferencesProvider).db;
+
+    return Container(
+      padding: const EdgeInsets.all(6.0),
+      color: color,
+      child: AdaptiveInkWell(
+        onTap: () async {
+          switch (openingDb) {
+            case OpeningDatabase.master:
+              return;
+            case OpeningDatabase.lichess:
+            case OpeningDatabase.player:
+              final gameId = GameId(game.id);
+              final archivedGame = await ref.read(
```

Comment:
This is not ideal but let's leave it like that for now. The `ArchivedGameScreen` should take either a `GameId` or `LightGameData` and load the game if it is a game id, like it is done on other screens.